src/Providers/Maps/Leaflet/Features/Shape.ts

```diff
@@ -0,0 +1,161 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace Provider.Maps.Leaflet.Feature {
+    export class Shape
+        implements
+            OSFramework.Maps.Feature.IShape,
+            OSFramework.Maps.Interface.IBuilder,
```

Comment:
I think we don't need to implement this!

Reply:
Done

Reply:
We only can remove the IDisposable, the other one needs to be implemented because of FeatureBuilder.